cmd/statusd/library.go

```diff
@@ -88,52 +92,24 @@ func ValidateNodeConfig(configJSON *C.char) *C.char {
 	return C.CString(string(respJSON))
 }
 
+//ResetChainData remove chain data from data directory
 //export ResetChainData
 func ResetChainData() *C.char {
 	_, err := statusAPI.ResetChainDataAsync()
 	return makeJSONResponse(err)
 }
 
+//CallRPC calls status node via rpc
 //export CallRPC
 func CallRPC(inputJSON *C.char) *C.char {
 	outputJSON := statusAPI.CallRPC(C.GoString(inputJSON))
 	return C.CString(outputJSON)
 }
 
-//export ResumeNode
-func ResumeNode() *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "ResumeNode")
-	return makeJSONResponse(err)
-}
-
-//export StopNodeRPCServer
-func StopNodeRPCServer() *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "StopNodeRPCServer")
-	return makeJSONResponse(err)
-}
-
-//export StartNodeRPCServer
-func StartNodeRPCServer() *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "StartNodeRPCServer")
-	return makeJSONResponse(err)
-}
-
-//export PopulateStaticPeers
-func PopulateStaticPeers() *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "PopulateStaticPeers")
-	return makeJSONResponse(err)
-}
-
-//export AddPeer
-func AddPeer(url *C.char) *C.char {
-	err := fmt.Errorf("%v: %v", common.ErrDeprecatedMethod.Error(), "AddPeer")
-	return makeJSONResponse(err)
-}
```

Comment:
Why have you removed all these?

Reply:
It's deprecated method,  Dead code linter pointed to some of them. I think i could to delete this deprecated method. Should i return them?

Reply:
Already answered below. Nope, it was good decision.